django-stubs/contrib/auth/tokens.pyi

```diff
@@ -10,7 +10,7 @@ class PasswordResetTokenGenerator:
     algorithm: str
     def make_token(self, user: AbstractBaseUser) -> str: ...
     def check_token(self, user: AbstractBaseUser | None, token: str | None) -> bool: ...
-    def _make_token_with_timestamp(self, user: AbstractBaseUser, timestamp: int, legacy: bool = ...) -> str: ...
+    def _make_token_with_timestamp(self, user: AbstractBaseUser, timestamp: int, secret: str | bool = ...) -> str: ...
```

Comment:
Does `bool` really make sense?

This gets passed to `salted_hmac()`, which calls `force_bytes(secret)`. Should it be `str | bytes` instead?

Reply:
I think you're right, I can't think of why I would've done `bool` here since it's the same type as https://github.com/typeddjango/django-stubs/pull/1411

Reply:
```suggestion
    def _make_token_with_timestamp(self, user: AbstractBaseUser, timestamp: int, secret: str | bytes = ...) -> str: ...
```